src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const lines = sourceString.split(';');
```

Comment:
After splitting the `sourceString` by semicolons, consider chaining `filter` to remove any empty or invalid lines before processing them. This would improve the robustness of your function against malformed input.